app/controllers/api/v0/manifests_controller.rb

```diff
@@ -12,30 +8,31 @@ def search
   end
 
   def validate
-    begin
-      manifest_content = JSON.parse(request.body.read)
+    manifest_content = read_body_as_json
+    unless performed?
```

Comment:
where does `performed?` come from in this context? I would guess it is related to an async process but we are not using those for API calls as far as I know so the naming is a bit confusing to me.